### DISCUSSION TITLE
How would I go about making everything bold?

### DISCUSSION THREAD

**Discussion by KorbinH** [akinomyoga/ble.sh#223] [Q&A] [answered]
I am sorry if you have answered this question before but I was using ble.sh and I was looking at how to configure the .blerc file and I got quickly confused on how I would go about making all of the text bold thank you.

**Comment by akinomyoga** [comment, marked as answer]
There is no single option to achieve that, but you need to add `bold` to all the faces listed by the command `ble-face`. I haven't carefully tested it, but maybe you can just modify the output of `ble-face` to add `bold` and then evaluate it by `eval`:

```bash
eval "$(ble-face | sed 's/$/,bold/')"
```

**Edit**: this is an improved version (see also https://github.com/akinomyoga/ble.sh/discussions/223#discussioncomment-3496922):

```bash
# blerc

eval "$(ble-face | sed -En '/=ref:|(^|,)bold(,|$)/!s/$/,bold/p')"
```

**Reply by KorbinH** [reply]
Thank you so much, using eval "$(ble-face | sed 's/$/,bold/')" in the .blerc file worked perfectly.

**Reply by akinomyoga** [reply]
I noticed that the previous code is not robust when a special type of face configuration (i.e., `facename1=ref:facename2`) is used, though it is not used within the default configuration. Here is an improved version:

```bash
# blerc

eval "$(ble-face | sed -En '/=ref:|(^|,)bold(,|$)/!s/$/,bold/p')"
```
